www/_posts/2019/2019-12-18-pyladies-global-council-process-vote.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post.html
+title: "PyLadies Global Council Selection Process Voting Now Open!"
+tags: [PyLadies, Global, Organizing]
+author: Lorena Mesa
+author_link: https://twitter.com/loooorenanicole
+---
+
+
+PyLadies Global Council Selection Process Voting Now Open!
+====================================================================================
+
+The PyLadies Global Interim team has now opened voting for PyLadies Chapter organizers to select one of three proposals for how the inaugural PyLadies Global Council election will function. 
+
+All information about voting can be found on the [PyLadies Global Organizing repository on issue #50](https://github.com/pyladies/global-organizing/issues/50).
+
```

Comment:
I'm wondering if it will be appropriate to add thanks/acknowledgement in this blog post to community members who have helped translated the proposals?

```suggestion
The proposals have been translated by volunteer community members to various languages: Arabic, French, Spanish, Portuguese, and Russian.
We'd like to thank Python community members for stepping up and for helping PyLadies in the translation effort.
(add names of translators here?)
```

Reply:
Yes that's a great suggestion! Thanks @Mariatta !